### ISSUE TITLE
Verification part

### ISSUE THREAD

**Issue by Shahnawazgrewal** [vdyashin/VoxCeleb#3]
Hi,

did you finish the verification part, if yes, what is the performance?

**Comment by v-iashin** [comment]
No, I didn't. I made a mistake somewhere, I believe. Could not find it back then.
